src/Sidebar/SidebarItem/SidebarItem.stories.js

```diff
@@ -54,6 +54,7 @@ Nested.args = {
     <SidebarItem
       key="withCount"
       {...WithCount.args}
+      // eslint-disable-next-line react/no-children-prop
```

Comment:
Rather than disable this warning, can you not update the code? Rather than passing children prop it should be possible to just explicitly create them by nesting.